pages/search.js

```diff
@@ -77,7 +87,7 @@ export default function Search({ data }) {
       setNotFound();
     }
 
-    setFilteredUsers(results);
+    setFilteredUsers(randomizeData(results));
```

Comment:
I think a simpler way might be to remove the custom function and use the `sort` function from javascript ...

```suggestion
    setFilteredUsers(results.sort(() => 0.5 - Math.random()));
```

Reply:
Wow, that's a great solution. Will make the changes right away!

Reply:
But you implemented it differently to the example I shared? 

You can just accept the "suggestion" if you agree with the suggestion